tests/fixtures/hello_world_linux/build/kconfigs.in

```diff
@@ -0,0 +1,4 @@
+source "/Users/alekseiapaseev/esp/esp-idf/components/esp_common/Kconfig"
```

Comment:
I think we only need the `project_description.json`, `hello_world.elf`, `config/sdkconfig.json`

Reply:
you may also remove the hello_world_main.c. since the functionality ("print hello world") should be easy to guess :D